test/isolation/expected/multi_transaction_indexing.out

```diff
@@ -0,0 +1,70 @@
+Parsed test spec with 7 sessions
+
+starting permutation: CI I1 Ic Bc P Sc
+step CI: CREATE INDEX test_index ON ts_index_test(location) WITH (hypertable.multi_transaction, hypertable.barrier_table='barrier'); <waiting ...>
+step I1: INSERT INTO ts_index_test VALUES (31, 6.4, 1);
+step Ic: COMMIT;
+step Bc: ROLLBACK;
+step CI: <... completed>
+step P: SELECT index_size FROM timescaledb_information.hypertable;
+index_size     
+
+64 kB          
+step Sc: COMMIT;
+
+starting permutation: I1 CI Bc Ic P Sc
+step I1: INSERT INTO ts_index_test VALUES (31, 6.4, 1);
+step CI: CREATE INDEX test_index ON ts_index_test(location) WITH (hypertable.multi_transaction, hypertable.barrier_table='barrier'); <waiting ...>
+step Bc: ROLLBACK;
+step Ic: COMMIT;
+step CI: <... completed>
+step P: SELECT index_size FROM timescaledb_information.hypertable;
+index_size     
+
+64 kB          
+step Sc: COMMIT;
+
+starting permutation: S1 CI Bc Sc P Ic
+step S1: SELECT * FROM ts_index_test;
+time           temp           location       
+
+1              23.4           1              
+11             21.3           2              
+21             19.5           3              
+step CI: CREATE INDEX test_index ON ts_index_test(location) WITH (hypertable.multi_transaction, hypertable.barrier_table='barrier'); <waiting ...>
+step Bc: ROLLBACK;
+step CI: <... completed>
+step Sc: COMMIT;
+step P: SELECT index_size FROM timescaledb_information.hypertable;
+index_size     
+
+48 kB          
+step Ic: COMMIT;
+
+starting permutation: F CI DI Bc P Ic Sc
+step F: SET client_min_messages TO 'fatal';
+step CI: CREATE INDEX test_index ON ts_index_test(location) WITH (hypertable.multi_transaction, hypertable.barrier_table='barrier'); <waiting ...>
+step DI: DROP INDEX test_index; <waiting ...>
+step Bc: ROLLBACK;
+step P: SELECT index_size FROM timescaledb_information.hypertable;
+index_size     
+
+0 bytes        
+step Ic: COMMIT;
```

Comment:
We don't have I1 do we really need Ic here?

Reply:
Unfortunately we either need to stop using `setup` block to start the transactions, or commit every transaction in every permutation, anything else and open transactions leak.